bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2.java

```diff
@@ -1259,6 +1279,9 @@ else if (v != null)
         v = Pio.getValue(paramSet, FPU_MASK_PROP);
         if (v != null) setFpuCustomMask(v);
 
+        v = Pio.getValue(paramSet, CUSTOM_SLIT_WIDTH_PROP);
+        if (v != null) setCustomSlitWidth(CustomSlitWidth.valueOf(v));
```

Comment:
In that case the default `Unknown` set during object initialization will be used. This covers the case of old programs that don't have this value.